recipes/python-neo/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "neo" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: python-{{ name }}
+  version: {{ version }}
+
+source:
+  fn: python-{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 98e7e8948158f4492fc94d9d44367c91
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.3.0
+    - quantities >=0.9.0
+
+  run:
+    - python
+    - numpy >=1.3.0
+    - quantities >=0.9.0
+
+test:
+  imports:
+    - neo
+    - neo.core
+    - neo.io
+    - neo.test
+    - neo.test.iotest
+
+about:
+  home: http://neuralensemble.org/neo
+  license: BSD 3-Clause
+  summary: Neo is a package for representing electrophysiology data in Python, together with support for reading a wide range of neurophysiology file formats.
```

Comment:
This is probably too long.  Can you try to keep to 80 characters for the summary?  We didn't get to that in the meeting on Friday, but that's one of our requirements for metadata, ensuring that your package will play nicely with Navigator.